app/Http/Controllers/TalksController.php

```diff
@@ -25,19 +25,21 @@ class TalksController extends BaseController
         'slides.url' => 'Slides URL must contain a valid URL',
     ];
 
-    protected $sorting_talks = [
+    protected $sorting_style = [
         'date' => '',
         'alpha' => '',
     ];
 
```

Comment:
I left it as strings  so that in the sortTalks method, it sets it to a classname. I also renamed it to represent that a little better. That way in the blade view, it looks like this.
```
<a href="{{ route('talks.index') }}?sort=alpha" class="{{ $sorting_style['alpha'] }}">Title</a>
```